mobile/lib/ui/settings/TEMP/lock_screen_option_password.dart

```diff
@@ -0,0 +1,179 @@
+import "package:flutter/material.dart";
+import "package:flutter/services.dart";
+import "package:photos/generated/l10n.dart";
+import "package:photos/theme/ente_theme.dart";
+import "package:photos/ui/common/dynamic_fab.dart";
+import "package:photos/ui/components/buttons/icon_button_widget.dart";
+import "package:photos/ui/components/text_input_widget.dart";
+import "package:photos/ui/settings/TEMP/lock_screen_option_confirm_password.dart";
+
+class LockScreenOptionPassword extends StatefulWidget {
+  const LockScreenOptionPassword({
+    super.key,
+    this.isAuthenticating = false,
+    this.authPass,
+  });
+  final bool isAuthenticating;
+  final String? authPass;
+  @override
+  State<LockScreenOptionPassword> createState() =>
+      _LockScreenOptionPasswordState();
+}
+
+class _LockScreenOptionPasswordState extends State<LockScreenOptionPassword> {
+  final _passwordController = TextEditingController(text: null);
```

Comment:
Make sure the controller is disposed.

Reply:
Does disposing this controller also throw an error/exception? If yes, mention it in a comment here.

Reply:
Yes this one also throws an exception